INSTALL.md

```diff
@@ -1,11 +1,17 @@
-# Install Zync
-Download this repository, install the dependencies and setup. These commands for MacOS are:
-```
-git clone git@github.com:3scale/zync.git
-cd zync
-brew bundle
-brew services start postgresql
-./bin/setup
-```
+# Install Zync (only explained for MacOS so far)
+1. Download this repository.
+`git clone git@github.com:3scale/zync.git`
+
+2. Move to the folder of the project.
+`cd zync`
+
+3. Install the dependencies. There is `Brewfile` containing all the dependencies.
+`brew bundle`
+
+4. Start postgres.
+`brew services start postgresql`
+
+5. Setup.
+`./bin/setup`
 
 The command `brew services start postgresql` starts the service of PostgreSQL. If `./bin/setup` aborts, make sure that this service is running. Verify with `brew services list` that has a status `started` and looking green. If the status is `started` but coloured orange, fix the errors indicated in the log located in `/usr/local/var/log/postgres.log`
```

Comment:
_...If `./bin/setup` aborts, make sure that the `PostgreSQL` service is running._

Reply:
If not, include the name of the service instead of "this".